src/wasm/wasm-binary.cpp

```diff
@@ -71,8 +71,9 @@ int32_t WasmBinaryWriter::writeU32LEBPlaceholder() {
   return ret;
 }
 
-void WasmBinaryWriter::writeResizableLimits(Address initial, Address maximum, bool hasMaximum) {
-  uint32_t flags = hasMaximum ? 1 : 0;
+void WasmBinaryWriter::writeResizableLimits(Address initial, Address maximum,
+                                            bool hasMaximum, bool shared) {
+  uint32_t flags = (uint32_t) hasMaximum | (uint32_t) shared << 1;
```

Comment:
I dunno, the point of having the bools in the signature is to abstract away the encoding details from the caller (as with initial and maximum). But having bits explicit is good; I moved that to `BinaryConsts`